policy/modules/apps/gpg.if

```diff
@@ -195,6 +195,97 @@ interface(`gpg_entry_type',`
 	domain_entry_file($1, gpg_exec_t)
 ')
 
+########################################
+## <summary>
+##	Execute the gpg_agent in the gpg_agent domain.
+## </summary>
+## <param name="domain">
+##	<summary>
+##	Domain allowed to transition.
+##	</summary>
+## </param>
+#
+interface(`gpg_agent_domtrans',`
+	gen_require(`
+		type gpg_agent_t, gpg_agent_exec_t;
+	')
+
+	corecmd_search_bin($1)
+	domtrans_pattern($1, gpg_agent_exec_t, gpg_agent_t)
+')
+
+########################################
+## <summary>
+##	Execute the gpg_agent in the caller domain.
+## </summary>
+## <param name="domain">
+##	<summary>
+##	Domain allowed access.
+##	</summary>
+## </param>
+#
+interface(`gpg_agent_exec',`
+	gen_require(`
+		type gpg_agent_exec_t;
+	')
+
+	corecmd_search_bin($1)
+	can_exec($1, gpg_agent_exec_t)
+')
+
+########################################
+## <summary>
+##	Execute gpg_agent in a specified domain.
+## </summary>
+## <desc>
+##	<p>
+##	Execute gpg_agent in a specified domain.
+##	</p>
+##	<p>
+##	No interprocess communication (signals, pipes,
+##	etc.) is provided by this interface since
+##	the domains are not owned by this module.
+##	</p>
+## </desc>
+## <param name="source_domain">
+##	<summary>
+##	Domain allowed to transition.
+##	</summary>
+## </param>
+## <param name="target_domain">
+##	<summary>
+##	Domain to transition to.
+##	</summary>
+## </param>
+#
+interface(`gpg_agent_spec_domtrans',`
```

Comment:
"spec_domtrans" means one using setexeccon instead of type_transition. This interface doesn't do that. Since it's not used, it could be dropped.

Reply:
I will remove it.